.github/ISSUE_TEMPLATE/improve-existing-docs.yml

```
@@ -0,0 +1,62 @@
+name: Improve existing docs
+about: Make a suggestion to improve our existing documentation.
+labels: content
+inputs:
+  - type: description
+    attributes:
+      value: |
+        HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
```

```suggestion
        HUBBERS BEWARE! Please confirm you're on the github/docs-internal repo and not the github/docs repo before proceeding, unless you want to make your work public to the entire internet. Open internal issues in gitthub/docs-content. The github/docs repo is open source and public to the entire internet.
```

---

@github/docs-content-strategy What do you think about this language to let Hubbers know that they should be in `docs-internal` for private PRs and `docs-content` for private issues?

---

I know you didn't ask me but I find the above confusing as it's mentioning three different repos. If I didn't have context I'd be confused about where to post.

---

```suggestion
        HUBBERS BEWARE! Please confirm you're in the right place. 
- Issues and PRs opened on github/docs will be visible to the world. It is open source and public to the entire internet.
- Open private PRs in github/docs-internal repo
- Open internal issues in gitthub/docs-content
```
Ooh, good call out. How about this?

---

So much better! One last question though, is there a reason you say "private PRs" but then "Internal Issues" rather than using the same word in both cases (Private or Internal)

---

Totally a mistake ✅

---

The latest suggestion looks great!